README.md

```diff
@@ -4,25 +4,122 @@ Anaconda Recipes
 This is a Continuum managed repository for recipes used in the
 free [Anaconda Python distribution](https://www.continuum.io/downloads).
 
-The machines (and additional software installed) to build these conda
-recipes are as follows:
+Building Conda Recipes
+================
 
-Linux:
-  * we use sepatate 32 and 64-bit machines
-  * CentOS release 5.11 (Final)
-  * gcc 4.4.7 (`yum install gcc44`, link `/usr/bin/gcc` to `gcc44`)
-  * GNU Fortran (GCC) 4.4.7
 
-Mac OS X (64-bit):
-  * Version 10.7.5
-  * gcc 4.2.1
-  * GNU Fortran (GCC) 4.2.3
+##### Linux (use separate 32 and 64-bit machines):
+
+Machine and additional software:
+ * CentOS release 5.11 
+ * gcc 4.4.7 (`yum install gcc44`, link `/usr/bin/gcc` to `gcc44`)
+ * GNU Fortran (GCC) 4.4.7
+
+We use a Docker image, “conda-builder”, to build the packages.
+
+Install Docker: 
+ 
+ Linux: 
+  * https://docs.docker.com/linux/step_one/ 
+  * Start the docker service: sudo service docker start
+ 
+Windows/Mac: 
+  * https://www.docker.com/products/docker-toolbox
+  * Run Docker Quickstart Terminal
+
+Download the following files: 
```

Comment:
We should provide a simple zip file, or perhaps roll the docker_wrapper.sh script into both start scripts (not as original source, but as a deployable step).  Having people download so many files is confusing.